libobs/audio-monitoring/pulse/pulseaudio-wrapper.c

```diff
@@ -54,36 +54,50 @@ void get_default_id(char **id)
 	} else {
 		*id = bzalloc(strlen(pdo->default_sink_name) + 9);
 		strcat(*id, pdo->default_sink_name);
-		strcat(*id, ".monitor");
 		bfree(pdo->default_sink_name);
 	}
 
 	bfree(pdo);
 	pulseaudio_unref();
 }
 
+/**
+ * Checks whether a the sound source (id1) is the .monitor device for the
+ *	selected monitoring output (id2).
+ */
```

Comment:
I'm sorry I should have mentioned this as well at the comment yesterday.
There should be just one article for a noun. A  space is preferable inside the comment.
```suggestion
/**
 * Checks whether the sound source (id1) is the .monitor device for the
 * selected monitoring output (id2).
 */
```

Reply:
Looked at that comment fifteen times, and never saw the typo. Thanks!